tests/core/test_context.py

```diff
@@ -1655,3 +1656,13 @@ def test_set_enable_error_table():
     # testing for False do not write error records in error table
     context.set_se_enable_error_table(False)
     assert context.get_se_enable_error_table is False
+
+
+def test_set_dq_rules_params():
+    # default case is True for enabling error table
+    context = SparkExpectationsContext(product_id="product1", spark=spark)
+    assert context.get_dq_rules_params == {}
+
+    # testing for False do not write error records in error table
```

Comment:
Please update the comments to appropriately reflect the test scenario

Reply:
updated comments properly